### PR TITLE
Root cleanup

### PR DESCRIPTION
**Pull request by latusaki**
- [x] Move example to  new directory
- [x] Remove RN project leftovers from root
- [x] Add npmignore to ignore example dir
- [x] Tested on iOS emulator
- [x] Tested on Android emulator

### CODE REVIEW

**Comment by jemise111** [comment]
Looks good! Couple questions/notes:

- Now that everything example-related is in the SwipeListExample directory do we still need the ios/ and android/ directories at the root level?
- Since the example won't be installed with npm install anymore can you add some documentation in the Example section of the readme with steps on how to run the example. Something like:

1. git clone... (since it's npm ignored)
2. cd SwipeListExample
3. npm install
4. react-native run-ios

that would be awesome. thanks!

**Comment by latusaki** [comment]
Will fix this weekend!

**Comment by latusaki** [comment]
The root folders for Android / iOS are removed already on my branch

**Comment by latusaki** [comment]
also, should the peer dependencies be bumped ? This works library works fine with RN 26 but npm shows warning for peer dependencies not satisfied.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ _**v0.1.0** introduced a breaking change if you had implemented the "Manually Cl
 npm install --save react-native-swipe-list-view
 ```
 
+## Running the example
+
+The application under ./SwipeListExample will product the above example. To run execute the following:
```
Comment (jemise111) [inline README.md]: product -> produce

**Comment by jemise111** [comment]
@latusaki thanks for updating, just fix the one small typo and we're good to go!

**Comment by jemise111** [comment]
Yeah good call, I'll take care of the peer dependencies issue on another branch

**Pull request merged**